FortranCon2021-fpm/README.md

```diff
@@ -0,0 +1,45 @@
+# FortranCon2021: fpm
+
+## Authors
+
+- [Sebastian Ehlert](https://github.com/awvwgk)
+- [Ondřej Čertík](https://github.com/certik)
+- [Milan Curcic](https://github.com/milancurcic)
+- [Philipp Engel](https://github.com/interkosmos)
+- [Jakub Jelínek](https://github.com/kubajj)
+- [Laurence Kedward](https://github.com/lkedward)
+- [Vincent Magnin](https://github.com/vmagnin)
+- [Arjen Markus](https://github.com/arjenmarkus)
+- [Emanuele Pagone](https://github.com/epagone)
+- [Ivan Pribec](https://github.com/ivan-pi)
+- [Brad Richardson](https://github.com/everythingfunctional)
+- [Damian Rouson](https://github.com/rouson)
+- [Carlos Une](https://github.com/brocolis)
+- [John Urban](https://github.com/urbanjost)
+- [Mark Wieczorek](https://github.com/MarkWieczorek)
+
+## Track
+
+Fortran-lang minisymposium (12 + 3 minutes discussion)
+
+## Presenter
+
+[Sebastian Ehlert](https://github.com/awvwgk)
+
+## Title:
+
+Fortran package manager
+
+## Abstract
+
+With the Fortran package manager (fpm) we are building the foundation of a new ecosystem for Fortran libraries and projects.
+The main goal of fpm is to create a reliable and productive tool and improve the user experience for Fortran programmers. 
+It provides an easy-to-use way to build applications and use libraries, making it especially easy to reuse projects as dependencies.
+Fpm takes care of fetching and building all transitive dependencies used to compile the main project.
```

Comment:
```suggestion
A language-specific package manager is an important element to build an efficient ecosystem of libraries and projects.
Fortran has long missed a dedicated package manager, making the user experience of practitioners cumbersome.
With the Fortran package manager (fpm) we are addressing these needs creating a reliable and productive tool that makes building applications and using libraries intuitive.
Fpm takes care of fetching and building all transitive dependencies used to build the main project making especially easy to reuse other projects.
```
The rationale of the above is to try catch the attention of the reader/audience focussing more on the motivation to develop fpm rather than "here is another project" (no criticism, I'm just exaggerating to make a point).